test/e2e/page/reasons-for-appealing/reasonsForAppealingOnePageForm.test.js

```diff
@@ -87,7 +105,9 @@ I see an error in the error summary`, async I => {
   I.see(content.listError);
 });
 
-Scenario(`When I add a reasons then click the add another reason button and enter the least amount 
+// this test is not passing.
+// Ticket was raised to fix it https://tools.hmcts.net/jira/browse/SSCS-3929
+xScenario(`When I add a reasons then click the add another reason button and enter the least amount 
```

Comment:
If this test is failing, does it not mean that the functionality of adding multiple reasons is broken?

Reply:
Yes, that's my understanding as well. I created this ticket for BA and business to prioritise https://tools.hmcts.net/jira/browse/SSCS-3929